gems/aws-sdk-iam/lib/aws-sdk-iam/customizations.rb

```diff
@@ -1,4 +1 @@
 # frozen_string_literal: true
-
-# customizations to generated classes
-require 'aws-sdk-iam/customizations/resource'
```

Comment:
Is this one missing a change?

Reply:
These are now included in the generated files (to account for the autoload).

Reply:
You can probably remove the file now if it's empty or at least the comment

Reply:
weirdly all services that don't have customizations have a completely empty customizations.rb file, so I just followed that.